lib/plugins/entities.js

```diff
@@ -446,6 +444,7 @@ function inject (bot, { version }) {
           player.displayName = new ChatMessage(JSON.parse(item.displayName))
         }
 
+        const playerEntity = Object.values(bot.entities).find(e => e.type === 'player' && e.username === item.name)
```

Comment:
Difference with previous code is this doesn't do .to lowercase()
But that's probably ok